TLM/TLM/State/Configuration.cs

```diff
@@ -186,12 +187,15 @@ public class LaneConnection : ISerializable {
 
             public bool sourceStartNode;
 
-            public bool Legacy => SerializableDataExtension.Version < 2;
+            public LaneEndTransitionGroup group = LaneEndTransitionGroup.Vehicle;
```

Comment:
I am concerned about continuing to introduce breaking file format changes. Can this all wait until the new persistence code is merged?

Reply:
This PR has waited for a month already. I don't think we need to wait more. conflicts will accumulate over time.
Is it breaking anything?

Reply:
it does not break anything for me. I made the connections bellow with TMPE from this PR then saved and loaded the game using stable TMPE:
![image](https://user-images.githubusercontent.com/26344691/172799090-3ffd85a8-5984-4004-a97a-5e007bb01f09.png)

Reply:
If we don't want to merge it with master, we should consider merging a feature branch instead of stopping this PR. but it is forward/backward compatible so I don't think we should.

Reply:
It's exactly the same situation as last time. If you save game with this build you won't be able to load it using current `master` or WS stable

Reply:
ok, did not throw exception or anything, loaded something, looks good. Ignore previous comment

Reply:
> loaded something

it is both forward/backward compatible. 
if you make different lane connections on mixed/car track lanes -> save -> load with earlier version of TMPE, then car/track lane connections will merge.

Reply:
yeah, noticed that, looks ok.